### DISCUSSION TITLE
Problema con Task1 del Lab4.01

### DISCUSSION THREAD

**Discussion by felipesanchez3196** [rramosp/ai4eng.v1#180] [04 - GESTIÓN DE DATOS]
Buenas tardes profesor y Felipe, con este lab tengo el problema que cuando envió mi código a la plataforma para que me califiqué, sale esto y no he encontrado algo para solucionarlo:

![image](https://user-images.githubusercontent.com/105797504/184011586-90c10b7b-9c00-4591-b011-da1c6feae667.png)

Acá abajo les dejo mi código para ver si tengo algo malo en el, y muchas gracias.

[LAB 04.01_task01.txt](https://github.com/rramosp/ai4eng.v1/files/9303738/LAB.04.01_task01.txt)

**Comment by Felipe-RA** [comment]
Saludos felipesanchez

Ese error te sale porque ya venció la fecha de entrega

Adicionalmente, para incluir tu código por favor hazlo con la funcionalidad de bloques de código, o compartiendo un enlace de google colab
